src/components/input/input.stories.mdx

```diff
@@ -33,3 +41,64 @@ You can also add `c-input` to `<select>` elements, in which case you’ll get a
     })}
   </Story>
 </Preview>
+
+## Elastic textarea
+
+Add `js-elastic-textarea` to a `<textarea>` element\* to have it expand as the user types more than will fit.
+
+If the `rows` attribute is set, it will be used as the minimum height.
+
+<p>
+  <small>* see also the JavaScript usage notes below</small>
+</p>
```

Comment:
I'm not sure how I feel about this footnote style. I wonder if we could add an inline link to below instead?

Reply:
Maybe I can rephrase the first sentence to something like "Add the `js-elastic-textarea` class to a `textarea` element (and follow the JS usage notes) to have it expand…"